### PR TITLE
EDU-241 Set up document blueprint

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-241

### CODE REVIEW

File: src/components/document-creation-modal.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  de: Neues Dokument
+copyTitleSuffix:
+  en: (copy)
+  de: (Kopie)
+copySlugSuffix:
+  en: copy
+  de: kopie
+titleRequired:
+  en: Please specify a document title
+  de: Bitte geben Sie einen Dokument Titel an
```
Comment (ahelmberger) [inline src/components/document-creation-modal.yml:12]: Dokumenttitel

File: src/components/document-creation-modal.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+useBlueprint:
+  en: Use blueprint
+  de: Blueprint benutzen
```
Comment (ahelmberger) [inline src/components/document-creation-modal.yml:15]: Vorlage benutzen

File: src/components/pages/settings.yml [hunk 3]
```diff
@@ -22,3 +22,6 @@ createDocumentRegenerationBatchHeader:
 createDocumentRegenerationBatchButton:
   en: Regenerate documents
   de: Dokumente neu erstellen
+blueprintDocumentHeader:
+  en: Blueprint document
+  de: Blueprint document
```
Comment (ahelmberger) [inline src/components/pages/settings.yml:27]: Maybe "Template for new documents" / "Vorlage für neue Dokumente"

File: src/components/document-creation-modal.yml (hunk 1 above)
Comment (adrianaluchian) [reply src/components/document-creation-modal.yml]: I tossed a coin between "Dokumenttitel", "Dokumentstitel" and "Dokument Titel" 🤦‍♀️ 
Thanks, corrected!

File: src/components/pages/settings.yml (hunk 3 above)
Comment (adrianaluchian) [reply src/components/pages/settings.yml:27]: Agreed, will rename `documentBlueprint` to `documentTemplate` in code as well.
Should we also rename the `blueprintKey` used so far in the cloning logic? I would like to do that.

File: src/components/document-creation-modal.yml (hunk 2 above)
Comment (adrianaluchian) [reply src/components/document-creation-modal.yml]: Changed ("Use default template" / "Standardvorlage benutzen")

**Review by ahelmberger — APPROVED** [review]

**Pull request merged**